backend/core/factories/user.py

```diff
@@ -0,0 +1,15 @@
+import factory
+
+from core.models import User
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+
+    class Meta:
+        model = User
+
+    email = factory.LazyAttribute(
+        lambda a: '{}.{}@example.com'.format(a.first_name, a.last_name).lower()
+    )
+    first_name = name = factory.Sequence(lambda n: 'first name %s' % n)
+    last_name = name = factory.Sequence(lambda n: 'last name %s' % n)
```

Comment:
Not essential but nice to have, we can use Faker providers to generate realistic values 
https://factoryboy.readthedocs.io/en/stable/index.html?highlight=first_name#realistic-random-values